packages/gamut/src/Form/Input.tsx

```diff
@@ -30,30 +33,40 @@ export type InputProps = InputHTMLAttributes<HTMLInputElement> & {
   required?: boolean;
   type?: string;
   valid?: boolean;
+  /**
+   * Allows Inputs to manage their own activated style state to acccount for some edge-cases.
+   */
+  activated?: boolean;
 };
-
 export interface StyledInputProps extends InputProps {
-  activated?: boolean;
   icon?: boolean;
 }
+
+/*
+ * @remarks We would love to properly type this with generics, but, alas, we cannot yet.
+ * @see https://github.com/Codecademy/client-modules/pull/270#discussion_r270917147
+ * @see https://github.com/Microsoft/TypeScript/issues/21048
+ */
 export interface InputWrapperProps extends InputProps {
-  as?: StyledComponent<
-    StyledInputProps,
-    React.DetailedHTMLProps<
-      InputHTMLAttributes<HTMLInputElement>,
-      HTMLInputElement
-    >
-  >;
+  as?: StyledComponent<StyledInputProps, React.PropsWithChildren<any>>;
   /**
    * A custom icon svg from gamut-icons.
    */
   icon?: typeof AlertIcon;
 }
 
+// ReactRecurly has some specific needs around padding and focus.
+export const reactRecurlyFormFieldFocusStyles = css`
+  ${formFieldFocusStyles}
+`;
+
+export const reactRecurlyFormFieldPaddingStyles = css`
+  ${formFieldPaddingStyles}
+`;
```

Comment:
```suggestion
export const reactRecurlyFormFieldFocusStyles = formFieldFocusStyles

export const reactRecurlyFormFieldPaddingStyles = formFieldPaddingStyles
```

I think we end up creating a className with `reactRecurlyFormFieldPaddingStyles-formFieldPaddingStyles` if we chain these.  Highly annoying TBH.

Reply:
oh fun! i'll swap it 🙏